01-getdata.md

```diff
@@ -151,3 +151,15 @@ if we get anything else, the response probably doesn't contain actual data
 > 
 > Read the [documentation](http://data.worldbank.org/developers/climate-data-api) for the Climate Data API,
 > and then write URLs to find the annual average temperature for Afghanistan between 1980 and 1999.
+
+> ## Follw up Multiple Choice Question {.callout}
```

Comment:
Please use .challenge as the style, and a meaningful title (e.g., "Defining REST API").